src/cnc/tests/test_provisioner.py

```diff
@@ -351,4 +348,135 @@ class AWSProvisionStageTwoServiceServerlessAndTwoResourceDynamoDBTest(
     def test_tf_is_valid(self):
         self.assertEqual(len(self.resources["aws_dynamodb_table"]), 2)
         self.assertEqual(len(self.resources["aws_lambda_function"]), 2)
+
+
+class AWSProvisionStageOneServiceServerlessOneResourceDynamoDBOneDatabaseTest(
+    AWSProvisionStageTestBase
+):
+    fixture_name = "serverless-1-service-1-dynamodb-1-db"
+    env_data_filepath = "environments_serverless_1_service_1_dynamodb_1_db.yml"
+
+    def test_tf_is_valid(self):
+        self.assertEqual(len(self.resources["aws_dynamodb_table"]), 1)
+        self.assertEqual(len(self.resources["aws_lambda_function"]), 1)
+        self.assertEqual(len(self.resources["aws_db_instance"]), 1)
+
+
+class AWSProvisionStageOneServiceServerlessOneDatabaseTest(AWSProvisionStageTestBase):
+    fixture_name = "serverless-1-service-1-db"
+    env_data_filepath = "environments_serverless_1_service_1_db.yml"
+
+    def test_tf_is_valid(self):
+        self.assertEqual(len(self.resources["aws_lambda_function"]), 1)
+        self.assertEqual(len(self.resources["aws_db_instance"]), 1)
+
+
+class AWSProvisionStageOneServiceServerlessTwoDatabaseTest(AWSProvisionStageTestBase):
+    fixture_name = "serverless-1-service-2-db"
+    env_data_filepath = "environments_serverless_1_service_2_db.yml"
+
+    def test_tf_is_valid(self):
+        self.assertEqual(len(self.resources["aws_lambda_function"]), 1)
+        self.assertEqual(len(self.resources["aws_db_instance"]), 2)
+
+
+class AWSProvisionStageOneServiceServerlessTwoResourceDynamoDBTwoDatabaseTest(
+    AWSProvisionStageTestBase
+):
+    fixture_name = "serverless-1-service-2-dynamodb-2-db"
+    env_data_filepath = "environments_serverless_1_service_2_dynamodb_2_db.yml"
+
+    def test_tf_is_valid(self):
+        self.assertEqual(len(self.resources["aws_dynamodb_table"]), 2)
+        self.assertEqual(len(self.resources["aws_lambda_function"]), 1)
+        self.assertEqual(len(self.resources["aws_db_instance"]), 2)
+
+
+class AWSProvisionStageOneServiceServerlessTwoResourceDynamoDBTwoDatabaseTest(
+    AWSProvisionStageTestBase
+):
+    fixture_name = "serverless-2-service-2-dynamodb-2-db"
+    env_data_filepath = "environments_serverless_2_service_2_dynamodb_2_db.yml"
+
+    def test_tf_is_valid(self):
+        self.assertEqual(len(self.resources["aws_dynamodb_table"]), 2)
+        self.assertEqual(len(self.resources["aws_lambda_function"]), 2)
+        self.assertEqual(len(self.resources["aws_db_instance"]), 2)
+
+
+class AWSProvisionStageOneBackendOneServerlessOneDatabaseTest(
+    AWSProvisionStageTestBase
+):
+    fixture_name = "backend-1-serverless-1-service-1-db"
+    env_data_filepath = "environments_backend_1_serverless_1_service_1_db.yml"
+
+    def test_tf_is_valid(self):
+        self.assertEqual(len(self.resources["aws_lambda_function"]), 1)
+        self.assertEqual(len(self.resources["aws_db_instance"]), 1)
```

Comment:
should assert that public IP not enabled, something about security groups or vpc routes as well